tools/automation.py

```diff
@@ -288,37 +288,46 @@ def get_new_mentors_in_yml_format(yml_file_path, xlsx_file_path, skiprows=1):
                 new_index += 1
                 mentors.append(mentor)
 
-        print(f"Added {len(mentors)} mentors to the mentors.yml file")
+        logging.info(f"Added {len(mentors)} mentors to the mentors.yml file")
     else:
-        mentors = get_all_mentors_in_yml_format(xlsx_file_path, skiprows)
+        mentors = get_all_mentors_in_yml_format(xlsx_file_path, skip_rows)
     return mentors
 
 
-def main():
+def run_automation():
     logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
-    if len(sys.argv) != 4:
-        logging.error("Usage: python SCRIPT_NAME FILE_PATH_MENTORS_XLSX FILE_PATH_MENTORS_YML MODE")
-        sys.exit(1)
-
-    xlsx_file_path = sys.argv[1]
-    yml_file_path = sys.argv[2]
-    mode = sys.argv[3]
-    # TODO: skiprows = sys.argv[4]
-    skiprows = 1
-
-    # Append new mentors to the current mentors.yml file
-    if mode == WriteMode.APPEND.value:
-        list_of_mentors = get_new_mentors_in_yml_format(yml_file_path, xlsx_file_path, skiprows)
+
+    if len(sys.argv) == 5:
+        xlsx_file_path = sys.argv[1]
+        yml_file_path = sys.argv[2]
+        mode = WriteMode(sys.argv[3])
+        skip_rows = sys.argv[4]
```

Comment:
It is needed to set type skip_rows to int since int is expected in the methods.
skip_rows = int(sys.argv[4])

Reply:
```suggestion
        skip_rows = sys.argv[4]
```
```suggestion
        skip_rows = int(sys.argv[4])
```